contracts/payloads/IGP83/PayloadIGP83.sol

```diff
@@ -0,0 +1,734 @@
+            msg.sender == PROPOSER ||
+                msg.sender == TEAM_MULTISIG ||
+                address(this) == PROPOSER_AVO_MULTISIG ||
+                address(this) == PROPOSER_AVO_MULTISIG_2 ||
+                address(this) == PROPOSER_AVO_MULTISIG_3 ||
+                address(this) == PROPOSER_AVO_MULTISIG_4 ||
+                address(this) == PROPOSER_AVO_MULTISIG_5,
+            "msg.sender-not-allowed"
+        );
+
+        uint256 totalActions = 1;
+        address[] memory targets = new address[](totalActions);
+        uint256[] memory values = new uint256[](totalActions);
+        string[] memory signatures = new string[](totalActions);
+        bytes[] memory calldatas = new bytes[](totalActions);
+
+        targets[0] = address(TIMELOCK);
+        values[0] = 0;
+        signatures[0] = "executePayload(address,string,bytes)";
+        calldatas[0] = abi.encode(ADDRESS_THIS, "execute()", abi.encode());
+
+        uint256 proposedId = GOVERNOR.propose(
+            targets,
+            values,
+            signatures,
+            calldatas,
+            description
+        );
+
+        require(proposedId == PROPOSAL_ID, "PROPOSAL_IS_NOT_SAME");
+    }
+
+    function execute() external {
+        require(address(this) == address(TIMELOCK), "not-valid-caller");
+        require(block.timestamp >= delaySetTime + delayTime, "delay not yet passed");
+
+        // Action 1: Readjust sUSDe-USDT<>USDT witdhrawal limit
+        action1();
+
+        // Action 2: Raise fUSDC and fUSDT Rewards Allowance
+        action2();
+
+        // Action 3: Set launch limits for USDC collateral vaults
+        action3();
+
+        // Action 4: Set launch limits for ezETH-ETH DEX and ezETH<>wstETH T1 & ezETH-ETH<>wstETH T2 vaults
+        action4();
+        
+        // Action 5: Update cbBTC-WBTC DEX configs
+        action5();
+
+        // Action 6: Update wstETH-ETH, weETH-ETH, rsETH-ETH DEX configs
+        action6();
+
+        // Action 7:Update USDC-USDT DEX limits
+        action7();
+
+        // Action 8: Set dust limits for USD0-USDC, fxUSD-USDC, USDC-BOLD DEX
+        action8();
+        
+    }
+
+    function verifyProposal() external view {}
+
+    /**
+     * |
+     * |     Team Multisig Actions      |
+     * |__________________________________
+     */
+    function setState(
+        uint256[] calldata actionsToSkip_
+    ) external {
+        if (msg.sender != TEAM_MULTISIG) {
+            revert("not-team-multisig");
+        }
+
+        for (uint256 i = 0; i < actionsToSkip_.length; i++) {
+            setActionSkippable[actionsToSkip_[i]] = true;
+        }
+    }
+
+     // Allows the team multisig to set a delay(max 5 days) for execution
+    function setDelayTime(uint256 _delayTime) external {
+        require(msg.sender == TEAM_MULTISIG, "not-team-multisig");
+        require(_delayTime <= 5 days, "delay exceeds 5 days");
+        delayTime = _delayTime;
+        delaySetTime = block.timestamp;
+    }
```

Comment:
all this now general logic should go into a inherited contract like PayloadIGPHelpers, less work for verification

Reply:
I see it' s done :D